Common/src/Common/Service/Data/Search/SearchTypeManager.php

```diff
@@ -23,4 +23,28 @@ class SearchTypeManager extends AbstractPluginManager
      * @var bool
      */
     protected $autoAddInvokableClass = false;
+
+    private array $config;
+
+    public function __construct(ContainerInterface $container, $config)
+    {
+        $this->config = $config;
```

Comment:
Do we need a config object? This is already a service manager so will already have registered factories/services.